lib/utils/thingMetadata.js

```diff
@@ -0,0 +1,16 @@
+import { ajax } from '../environment';
+import { batch } from './async';
+
+export const getPostMetadata = batch(async requests => {
+	const byId = requests.map(r => r.id)
+		.map(id => `t3_${id}`).join(',');
+
+	const { data: { children } } = await ajax({
+		url: `/by_id/${byId}.json`,
+		type: 'json',
+	});
+
+	return children.reduce((map, { data }) => map.set(data.id, data), new Map());
+}, { size: 50 });
```

Comment:
If you add `?limit=100` to the url you should be able to make this `100`, [e.g.](https://www.reddit.com/by_id/t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g,t3_4r2r9g.json?limit=100)